start_depth: 1.3
initial_depth: 1.3
initial_thrust: 0.18
timeout_init: 30
timeout_lane_search: 10
timeout_fix_lane: 10

timeout_position: 20
fix_by_position: true
real_lane_width: 0.15
fix_p: 0.2
fix_d: 0.6

eps_distance: 0.04
eps_angle: 5
success_time: 5
lanes_needed: 3